src/types.ts

```diff
@@ -69,6 +69,11 @@ export interface BootstrapConfig {
     featureFlagPayloads?: Record<string, JsonType>
 }
 
+export interface PerformanceCaptureConfig {
+    replay?: boolean
```

Comment:
What does replay performance include? I'm guessing it's network performance. Maybe it could have a more explicit name to that effect

Reply:
renamed this which ended up forcing m to write tests to improve the existing behaviour anyway